apps/container/src/components/ConceptsHorizontalChart.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <div class="grading">
+    <el-tooltip
+      v-for="grade in orderedDistribution"
+      placement="top"
+      :key="grade.conceito"
+      :hide-after="0"
+      :content="`
+        ${grade.conceito}: ${grades[grade.conceito].toFixed(1)}% (${
+          grade.count
+        } notas)`"
+    >
+      <span
+        class="grading-segment"
+        :class="grade.count < unthrustableThreshold ? 'unthrustable' : ''"
+        :style="{
+          background:
+            conceptsColor[grade.conceito as keyof typeof conceptsColor],
+          width: `${grades[grade.conceito]}%`,
+        }"
+      >
+      </span>
+    </el-tooltip>
+
+    <span
+      v-if="gradeData.count < unthrustableThreshold"
+      class="low-samples text-body-2"
+      >Dados sem muitas amostras</span
+    >
+  </div>
+</template>
+
+<script setup lang="ts">
+import { conceptsColor } from 'consts';
+import { ConceptData, SubjectSpecific } from 'types';
+import { transformConceptDataToObject } from 'utils';
+import { computed, PropType } from 'vue';
+
+const props = defineProps({
+  gradeData: { type: Object as PropType<SubjectSpecific>, required: true },
+});
+
+const unthrustableThreshold = 10;
+
+const orderedDistribution = computed(() => {
+  const distribution: ConceptData[] = JSON.parse(
+    JSON.stringify(props.gradeData.distribution),
+  );
```

Comment:
se a ideia é fazer um deep copy de um array, não seria melhor usar isso?
```suggestion
  const distribution: ConceptData[] = [...props.gradeData.distribution]
```

Reply:
JSON.parse e JSON.stringify são pesados, só faz sentido fazer isso se ele não conseguir copiar os objetos que estão dentro do array e vc for mutar eles, precisa testar